src/components/Layout.test.tsx

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import { mount } from 'enzyme';
+import 'jsdom-global/register';
+import Head from 'next/head';
+import Layout from './Layout';
+
+
```

Comment:
```suggestion
```

Same here, eslint [no-multiple-empty-lines](https://eslint.org/docs/rules/no-multiple-empty-lines)

Reply:
I already add the rule to my linter, I'll commit the changes.